### PR TITLE
add validate for blacklist

### PR DESCRIPTION
**Pull request by juniuszhou**

### CODE REVIEW

File: x/ethbridge/types/msgs.go [hunk 1]
```diff
@@ -671,6 +672,20 @@ func (msg MsgSetBlacklist) GetSignBytes() []byte {
 }
 
 func (msg *MsgSetBlacklist) ValidateBasic() error {
+	if msg.From == "" {
```
Comment (smartyalgo) [inline x/ethbridge/types/msgs.go:675]: Should we also check if it is valid bech32 address?
`sdk.AccAddressFromBech32(msg.From)`

File: x/ethbridge/types/msgs.go [hunk 2]
```diff
@@ -671,6 +672,20 @@ func (msg MsgSetBlacklist) GetSignBytes() []byte {
...
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, msg.From)
+	}
+
+	if len(msg.Addresses) == 0 {
+		return ErrEmptyBlackList
```
Comment (smartyalgo) [inline x/ethbridge/types/msgs.go:680]: Why isnt an empty blacklist allowed? For example, when we init, or when we want to clear the list

File: x/ethbridge/types/msgs.go (hunk 1 above)
Comment (juniuszhou) [reply x/ethbridge/types/msgs.go:675]: will add it.

File: x/ethbridge/types/msgs.go (hunk 2 above)
Comment (juniuszhou) [reply x/ethbridge/types/msgs.go:680]: It is my wrong assumption, I thought it is an incremental function. will remove this check.

Comment (smartyalgo) [reply x/ethbridge/types/msgs.go]: Can you add a test such to verify empty list wont throw err?

Comment (juniuszhou) [reply x/ethbridge/types/msgs.go]: sure, will add it.
